api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/HawkularMetricsRestApp.java

```diff
@@ -110,7 +111,8 @@ public HawkularMetricsRestApp() {
         classes.add(EmptyPayloadFilter.class);
         classes.add(TenantFilter.class);
         classes.add(MetricsServiceStateFilter.class);
-        classes.add(CorsFilter.class);
```

Comment:
If in the JAX-RS 1.1 impl the feature is implemented as a servlet filter, then there's no need to declare it here. Only JAX-RS related classes should be added to `classes`.